apps/site/assets/ts/schedule/components/ScheduleDirection.tsx

```diff
@@ -104,11 +104,12 @@ const ScheduleDirection = ({
 
   // If the route is a rail type, any other routePatterns
   // of the current typicality are a branch
-  const branchPatterns = !isABusRoute(route)
-    ? routePatternsInCurrentDirection.filter(
-        pattern => pattern.typicality === defaultRoutePattern.typicality
-      )
-    : [];
+  const branchPatterns =
+    !isABusRoute(route) && !isACommuterRailRoute(route)
```

Comment:
But Hannah! A few of the commuter rail routes _do_ have branches! 🙈 (localhost below!) 
![image](https://user-images.githubusercontent.com/2136286/112515551-bbb33380-8d6c-11eb-9092-032f52231642.png)

Reply:
Ohhh nooooooooo 😭

Reply:
Fixed 😅